__tests__/astToPlainText.test.js

```diff
@@ -2,21 +2,21 @@ import { hast, astToPlainText } from '../index';
 
 describe('astToPlainText()', () => {
   it("converts br's to ''", () => {
-    const txt = `<br>`;
```

Comment:
Our ESLint config now prevents using templated strings on strings that aren't actually templates.